sdk/communication/Azure.Communication.CallAutomation/tests/CallMedias/CallMediaTests.cs

```diff
@@ -125,7 +156,11 @@ public void MediaOperations_Return404NotFound(Func<CallMedia, Response> operatio
                 },
                 new Func<CallMedia, Task<Response>>?[]
                 {
-                   callMedia => callMedia.StartRecognizingAsync(_fullRecognizeOptions)
+                   callMedia => callMedia.StartRecognizingAsync(_dmtfRecognizeOptions)
+                },
+                new Func<CallMedia, Task<Response>>?[]
+                {
+                   callMedia => callMedia.StartRecognizingAsync(_ivrChoiceRecognizeOptions)
```

Comment:
Nit: should we remove "Ivr" text from all places to keep it consistent with server side?